src/components/FooterDesktop.tsx

```diff
@@ -46,6 +46,18 @@ const FooterText = styled(Typography)({
   color: theme.palette.text.primary
 });
 
+const FooterLink = styled(Link)({
```

Comment:
did you want these style changes in the same PR?

Reply:
This one is for this issue: https://github.com/dOrgTech/landing-page-v2/issues/222